.github/workflows/on-push-to-release-branch.yaml

```diff
@@ -65,13 +65,17 @@ jobs:
             rm -rf bin
             VERSION="${{ needs.release.outputs.version }}"
             echo "version: ${VERSION}"
-            dotnet build --configuration Release -f netstandard2.0 -p:DefineConstants=USE_GRPC_WEB
-            DLL_FILE=./bin/Release/netstandard2.0/Momento.Sdk.dll
-            ARCHIVE_FILE_NAME=Momento.Sdk.Unity.${VERSION}.dll
+            dotnet publish --configuration Release -f netstandard2.0 -p:DefineConstants=USE_GRPC_WEB
```

Comment:
```suggestion
            dotnet build --configuration Release -f netstandard2.0 -p:DefineConstants=USE_GRPC_WEB
```

Reply:
I think it needs to be publish to gather the dlls into `./bin/Release/netstandard2.0/publish/`

Reply:
Gotcha! Sorry, I just glanced at this yesterday, obviously without paying enough attention :-)